### ISSUE TITLE
End-to-End Blind Image Quality Assessment Using Deep Neural Networks

### ISSUE THREAD

**Issue by ghuijo** [yskim0/One-Day-One-Paper#63]
~~메디컬 기반 prediction 논문만 계속 보니 너무 힘들다...~~
이번 주말동안은 IQA(Image Quality Assessment) 논문 리딩.

## Abstract & Introduction
* MEON (multi-task end-to-end optimized deep neural network) 제안.
* BIQA (blind image quality assessment)를 위한 모델임.
* 기존 방법들과 다르게 training process를 2단계로 나눠서 진행.
  * distortion type identification sub-network를 train시킴. (large training samples)
  * pre-trained된 초기 layer에서 시작해서 first sun-network의 output까지 quality prediction sub-network을 train시킴. (다양한 gradient descent 방법 사용)
 * rectified linear unit 대신에 activation function으로 GDN (biologically inspired generalized divisive normalization) 사용.
 * 이미지 내 존재하는 왜곡의 유형과 정도를 동시에 예측해내는 NR IQA 모델.

## MEON for BIQA
* 256x256x3 image를 input으로 집어넣고 perceptual quality score를 예측함.

##
* MEON 구조
<image src="https://user-images.githubusercontent.com/67627471/129448072-1b785990-47ff-4323-a231-00eb53f052c9.png" width="80%" height="80%" />

* multi-task learning 구조
<image src="https://user-images.githubusercontent.com/67627471/129448144-2c4e75bc-04c9-4aa5-8145-4864e58803cf.png" width="30%" height="30%" />

##
* 왜곡 이미지는 4층의 Conv layer를 거친 후에 먼저 sub Network1에 입력되어 왜곡의 유형 예측.
* 4층의 conv layer를 거쳐 얻은 특성정보는 sub network1에서 얻은 왜곡 유형과 함께 sub network2에 입력되고 최종적으로 품질 점수 예측됨.
* Shared layers는 subNetwork1과 함께 pre-train됨.
* pre-training의 learning rate은 10^-2로 시작 ~ 10^-4까지.
* subNetwork2를 train시킬 때 전체 네트워크의 가중치들이 최적화됨.
* 최적화될 때 learning rate은 10^-4로 고정.

##
* GDN ( Generalized Divisive Normalization) 을 activation function으로 사용.
* GDN 정의
<image src="https://user-images.githubusercontent.com/67627471/129448040-16ae6e51-1f51-418c-a167-04bcafe8a36e.png" width="40%" height="40%" />

##
* ADAM 최적화 방법을 사용.
* 4개의 왜곡 유형(JP2k, JPEG, WN, Blur)을 가진 이미지들로만 train, test 수행.

### loss 식
* Subtask I에서의 empirical cross entropy loss
<image src="https://user-images.githubusercontent.com/67627471/129448431-d7b56e9c-92ac-499b-b19c-ad260c6f38ce.png" width="50%" height="50%" />

* Subtask II에서의 l1-norm (역시 empirical loss function으로)
<image src="https://user-images.githubusercontent.com/67627471/129448461-f069cf7c-cd60-4558-b2c7-cc7ecceed5ea.png" width="50%" height="50%" />

* 총 loss는 l2에 가중치 곱해서 서로 더함.
<image src="https://user-images.githubusercontent.com/67627471/129448483-0f862f56-5c58-4e03-96a9-b34972d28229.png" width="40%" height="40%" />

## 결과
* MEON을 기존 BIQA model들(DIIVINE, BRISQUE, CORNIA, ILNIQD, BLISS, HOSA, dipIQ, deepIQA 등)과 비교함. (4개의 distortion types에 대해서; LIVE, CSIQ, TID2013, Waterloo Exploration Database)

* CSIQ
<image src="https://user-images.githubusercontent.com/67627471/129448731-89f728ba-3662-430e-a034-d1b27a33608e.png" width="40%" height="40%" />

* TID2013
<image src="https://user-images.githubusercontent.com/67627471/129448743-e9afb8e2-46ed-4518-82c5-610d99d86fea.png" width="40%" height="40%" />

* confusion matrices의 distortion type 예측(CSIQ, TID2013, Waterloo~)
<image src="https://user-images.githubusercontent.com/67627471/129448716-7a5dc2d8-5bb7-4664-a4a0-850c432ec60a.png" width="40%" height="40%" />

##
* 식이 너무 많을 때 어떻게 하시나요?? 다 이해하려고 하니까 너무 괴롭고 시간도 오래 걸리네요...
